### ISSUE TITLE
css-orientation-lock rule should support CSS rotate

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#3920]
Our ACT nightly tests started failing after the CSS orientation rule added examples with the CSS `rotate` prop:

https://app.circleci.com/pipelines/github/dequelabs/axe-core/4921/workflows/53584a64-11a1-4bb5-bd24-872f02c1cece/jobs/54868

The following example should fail axe-core:

```html
<html lang="en">
<head>
  <style>
    @media (orientation: portrait) {
      html { rotate: 90deg; }
    }
  </style>
</head>
<body>
  Page Content
</body>
</html>
```

**Comment by WilcoFiers** [comment]
Here's the PR that changes this in ACT: https://github.com/act-rules/act-rules.github.io/pull/1742/files

**Comment by padmavemulapati** [comment]
Valdiated with the latest axe-core develop branch code base, with the below code snippet:
```

  <p id="fail1" class="fail1">This is my first paragraph with this ID.</p>
<div style="display: none">
  <p id="fail1">This is my second paragraph with this ID.</p>
</div>
<head>
  <style>
    @media (orientation: portrait) {
      html { rotate: 90deg; }
    }
  </style>
</head>
<body>
  Page Content
</body>
```
`css-orientation-lock` will fail 


![Image](https://user-images.githubusercontent.com/47409579/230264544-a4367d95-e54a-469c-8583-c36cc44c998e.png)
